db/models.py

```diff
@@ -1,5 +1,9 @@
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
 
+from settings import AUTH_USER_MODEL
```

Comment:
The import of `AUTH_USER_MODEL` from `settings` is incorrect. You should import it from `django.conf` using `from django.conf import settings` and then use `settings.AUTH_USER_MODEL` in the `ForeignKey` definition.